tests/integration/pdo/mysql/base-class/test_query_1.php

```diff
@@ -0,0 +1,52 @@
+<?php
```

Comment:
Similar to the `port` vs `socket` naming, can we name the `query` tests to reflect the differences in the query being tested? For example, `_arg`, `_fetch_column`, `_fetch_class`, `_fetch_into`?

Reply:
@bduranleau-nr ce4c0fa51733fc493cbf2a1ee6e14826c7ecca6f?